### PR TITLE
Supp app content changes for rev3 DAH-38

### PR DESCRIPTION
**Pull request by akegan**
DAH-38

Updated some of the content to match the [figma mock-up](https://www.figma.com/file/FFaD5ZlsL9QzpTsA8yP7iI/scooting-partners-supp-app-170930954?node-id=167%3A0)

### CODE REVIEW

File: app/javascript/components/supplemental_application/SupplementalApplicationContainer.js [hunk 1]
```diff
@@ -31,8 +31,8 @@ const ConfirmedPreferencesSection = ({
   visited
 }) => (
   <ContentSection
-    title='Confirmed Preferences and Priorities'
-    description='Please allow the applicant 24 hours to provide appropriate preference proof if not previously supplied.'
+    title='Preferences and Priorities'
+    description={<> Complete this section first. <b>You must confirm claimed preferences before sending out a post-lottery letter. </b> Please allow the applicant 24 hours to provide appropriate preference proof if not previously supplied.</>}
```
Comment (james-wills-sf) [inline app/javascript/components/supplemental_application/SupplementalApplicationContainer.js]: nit: space before C in `<> Complete`, unsure if this shows up. Also does `letter. </b> Please` result in a double space after the period?

Also, are the `<></>` necessary here? I didn't realize `sometext <b> bold text </b> othertext` results in an array of nodes, but I guess that makes sense.

File: app/javascript/components/supplemental_application/SupplementalApplicationContainer.js [hunk 2]
```diff
@@ -80,15 +86,18 @@ const LeaseInformationSection = ({ form, submitting, values, visited }) => (
...
       <RentalAssistance form={form} submitting={submitting} />
     </ContentSection.Sub>
   </ContentSection>
 )
 
 const DemographicsSection = () => (
-  <ContentSection title='Demographics'>
+  <ContentSection
+    title='Demographics'
+    description='FInish up by completing this section once a lease is signed. This information is required to track dependents, seniors, and minors in households that have obtained housing through MOHCD programs.'
```
Comment (james-wills-sf) [inline app/javascript/components/supplemental_application/SupplementalApplicationContainer.js]: Typo: `FInish`

**Review by james-wills-sf — APPROVED** [review]
@akegan lgtm

File: app/javascript/components/supplemental_application/SupplementalApplicationContainer.js (hunk 1 above)
Comment (akegan) [reply app/javascript/components/supplemental_application/SupplementalApplicationContainer.js]: doesn't compile if I remove them and do:
```
description={Complete this section first. <b>You must confirm claimed preferences before sending out a post-lottery letter.</b> Please allow the applicant 24 hours to provide appropriate preference proof if not previously supplied.}
```
If I put it in quotes the bold text doesn't render as bold

Comment (james-wills-sf) [reply app/javascript/components/supplemental_application/SupplementalApplicationContainer.js]: okay! I guess it does turn into an array

**Pull request merged**